spec/paratrooper/source_control_spec.rb

```diff
@@ -6,12 +6,25 @@
   describe "remote" do
     it "returns string of git representing remote repo" do
       config = instance_double(Paratrooper::Configuration,
-        deployment_host: 'HOST', app_name: 'APP'
+        deployment_host: 'HOST', app_name: 'APP', remote_name: 'APP'
```

Comment:
Looks like you have a test below covering the explicit remote_name, I would revert this scenario back to the original.

Reply:
See https://github.com/mattpolito/paratrooper/pull/87#discussion_r446202882 please.